packages/parser/src/language/packet/packet.langium

```diff
@@ -0,0 +1,14 @@
+grammar Packet
+import "../common/common";
+
+entry Packet:
+	"packet-beta" NEWLINE*
+  TitleAndAccessibilities?
```

Comment:
I assume you took this from `info.langium`, it would be fine for `info` diagram, but other diagram it's not.

If we used this approach then the following code would be valid:

```
packet-beta 1: "block"
```

And you messed the `NEWLINE` at the beginning.

```
  NEWLINE*
  "packet-beta"
  (
    NEWLINE* TitleAndAccessibilities
    | NEWLINE+ blocks+=PacketBlock*
    | NEWLINE*
  )
```